### PR TITLE
Implement multiple response questions

### PR DESCRIPTION
**Pull request by lowjoel**

### CODE REVIEW

File: app/models/course/assessment/answer/multiple_response.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Course::Assessment::Answer::MultipleResponse < ActiveRecord::Base
+  acts_as :answer, class_name: Course::Assessment::Answer.name, inverse_of: :actable
+
+  has_many :options, class_name: Course::Assessment::Answer::MultipleResponseOption.name,
+                     dependent: :destroy, foreign_key: :answer_id, inverse_of: :answer
+
+           accepts_nested_attributes_for :options
```
Comment (allenwq) [inline app/models/course/assessment/answer/multiple_response.rb]: Wrong indentation

**Comment by allenwq** [comment]
@lowjoel I think you need run rubocop locally since hound is not working. 

Despite of this, this PR looks good to me.

**Comment by lowjoel** [comment]
@allenwq "apart from this" :stuck_out_tongue:

thanks, I've rubocop'ed locally...

**Comment by allenwq** [comment]
Thanks !

**Pull request merged**